server/app/app.py

```diff
@@ -1,26 +1,189 @@
 import os
 import datetime
 
-from flask import Flask
+from flask import Flask, redirect, current_app, make_response, render_template
+from flask import Blueprint, request, send_from_directory
 from flask_caching import Cache
 from flask_compress import Compress
 from flask_cors import CORS
+from flask_restful import Api, Resource
 
-from server.app.rest_api.rest import get_api_resources
-from server.app.util.utils import Float32JSONEncoder
-from server.app.web import webapp
+from http import HTTPStatus
+
+import server.common.rest as common_rest
+from server.common.utils import path_join, Float32JSONEncoder
+from server.data_common.matrix_loader import MatrixDataLoader, MatrixDataType, MatrixDataCacheManager
+
+from functools import wraps
+
+webbp = Blueprint("webapp", "server.common.web", template_folder="templates")
+
+
+@webbp.route("/")
+def index():
```

Comment:
I'm not sure this is the right action to take in dataroot mode.  More likely configurations will want a redirect to some other location.

That said, for a CLI multi-dataset mode, it makes sense to me.

Lets leave it as is for this PR, as long as --dataroot is hidden option.   We'll sort it out more as we continue in future PRs and deal with AWS/EB deployment.